src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,214 @@
+      opacity: 1;
+      pointer-events: auto;
+    `}
+`
+
+const DropArea = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  border-radius: ${radius.lg};
+  padding-block-start: ${space[16]};
+  padding-block-end: ${space[24]};
+  padding-inline: ${space[16]};
+  text-align: center;
+  position: relative;
+  background-color: ${color.elevatedBackground};
+
+  @media ${device.tablet} {
+    background-color: ${color.actionBackground};
+    border: 2px dashed ${color.earth};
+  }
+`
+
+const Illustration = styled.img`
+  cursor: pointer;
+`
+
+const Title = styled(H4)`
+  font-weight: ${fontWeight.semiBold};
+`
+
+const Information = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding-block-start: ${space[12]};
+  flex: 1;
+  gap: ${space[8]};
+`
+
+const FileInput = styled.input`
+  display: none;
+`
+
+const DesktopUI = styled.div`
+  display: none;
+
+  @media ${device.tablet} {
+    display: block;
+  }
+`
+
+const MobileUI = styled.div`
+  display: block;
+  width: 100%;
+
+  @media ${device.tablet} {
+    display: none;
+  }
+`
+
+const StyledButton = styled(Button)`
+  width: 100%;
+`
+
+export const Dropzone = ({
+  title,
+  subtitle,
+  action,
+  mobileAction,
+  dropTitle,
+  accept = ['*'],
+  multiple = false,
+  ariaLabel,
+  onFileChange,
+  onUnacceptedFileChange,
+}: DropzoneProps) => {
+  const inputFile = useRef<HTMLInputElement | null>(null)
+  const [isDragging, setDragging] = useState(false)
+
+  const onButtonClick = () => inputFile.current?.click()
+  const onDragOver = (e: DragEvent<HTMLDivElement>) => e.preventDefault()
+  const onDragLeave = () => setDragging(false)
+  const onDragExit = () => setDragging(false)
+
+  const onDragEnter = (e: DragEvent<HTMLDivElement>) => {
+    e.preventDefault()
+    setDragging(true)
+  }
+
+  const onDrop = (e: DragEvent<HTMLDivElement>) => {
+    e.preventDefault()
+    setDragging(false)
+
+    if (
+      accept[0] !== '*' &&
+      Array.from(e.dataTransfer.files).some(
+        ({ name }) => !accept.includes(`.${name.split('.')[1]}`)
+      )
+    ) {
+      if (onUnacceptedFileChange) {
+        onUnacceptedFileChange()
+      }
+
+      return
+    }
+
+    multiple
+      ? onFileChange(e.dataTransfer.files)
+      : onFileChange(e.dataTransfer.files[0])
+  }
+
+  const onSelectFile = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e?.target?.files?.length) {
+      multiple ? onFileChange(e.target.files) : onFileChange(e.target.files[0])
+    }
+  }
+
+  return (
+    <DropArea data-testid="droparea" onDragEnter={onDragEnter}>
+      <DropOverlay
+        data-testid="dropoverlay"
+        isDragging={isDragging}
+        onDragOver={onDragOver}
+        onDragLeave={onDragLeave}
+        onDragExit={onDragExit}
+        onDrop={onDrop}
+      >
+        <img src={ILLUSTRATION} alt={dropTitle} />
+        <h2>{dropTitle}</h2>
```

Comment:
Same as the h4 comment

Reply:
Looking at other components that aren't specifically made for holding text (e.g. `Card`) I think it's okay to make assumptions about the hierarchy on the page. But let me know if you think otherwise @TicketSwap/frontend I'm also okay with adding the prop.